iesurveychecks/google-functions.md

```diff
@@ -0,0 +1,16 @@
+# Instructions on Google Sheet functions
+
+The graphs and tables in the HFC template are created mainly using the following functions:  
```

Comment:
the numbering on this seems funny -- it looks to me like some should be sub-bullets, e.g 8 and 9 seem like they are instructions for how to implement 7